docs/sources/tempo/operations/dedicated_columns.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Dedicated columns
+description: Learn how to use dedicated columns to improve query performance.
+weight: 42
+---
+
+# Dedicated columns
+
+Dedicated columns are a way to improve query performance by storing the most frequently used columns in dedicated columns,
```

Comment:
```suggestion
Dedicated columns improve query performance by storing the most frequently used columns in dedicated columns,
```

Reply:
Is this only for TraceQL queries or other queries?

Reply:
>Saying they improve query performance is more direct, but is it accurate for every case?

Good point. It either improves query performance or keeps it the same. IMO, how it's worded now it's accurate. TraceQL queries that use attributes are expected to always run better with dedicated columns.

>Is this only for TraceQL queries or other queries?

It affect TraceQL queries that use attributes in them—eg. `{span.storage.format = "vparquet3"}`. Queries that use intrinsics or special attributes are unaffected—eg. `{name = "foo"}`.